Frameworks/include/UIViewInternal.h

```diff
@@ -33,6 +33,27 @@
 @class WXFrameworkElement;
 @class WUXIPointerRoutedEventArgs;
 
+// Round subpixel values to be able to perform per-pixel UI placement/calculations
+inline float doPixelRound(float f) {
```

Comment:
maybe, plus you are looking at a label proxy right?  the layout code is different for the proxies.

---
In reply to: [100634412](https://github.com/Microsoft/WinObjC/pull/1920#discussion_r100634412) [](ancestors = 100634412)